jaxrs-api/src/main/java/jakarta/ws/rs/SeBootstrap.java

```diff
@@ -140,6 +144,70 @@
         return RuntimeDelegate.getInstance().bootstrap(application, configuration);
     }
 
+    /**
+     * Starts the provided application using a default configuration.
+     *
+     * <p>
+     * This method is intended to be used in Java SE environments only. The outcome of invocations in Jakarta EE container
+     * environments is undefined.
+     * </p>
+     *
+     * @param application The application to start up.
+     * @return {@code CompletionStage} (possibly asynchronously) producing handle of the running application
+     * {@link SeBootstrap.Instance instance}.
+     * @see Configuration
+     * @since 3.1
+     */
+    static CompletionStage<Instance> start(final Application application) {
+        Configuration configuration = Configuration.builder().build();
+        return start(application, configuration);
+    }
+
+    /**
+     * Starts the provided application using the specified configuration. Creates application instance
+     * from class using default constructor. Injection is not supported.
+     *
+     * <p>
+     * This method is intended to be used in Java SE environments only. The outcome of invocations in Jakarta EE container
+     * environments is undefined.
+     * </p>
+     *
+     * @param clazz The application class.
+     * @param configuration Provides information needed for bootstrapping the application.
+     * @return {@code CompletionStage} (possibly asynchronously) producing handle of the running application
+     * {@link SeBootstrap.Instance instance}.
+     * @see Configuration
+     * @since 3.1
+     */
+    static CompletionStage<Instance> start(final Class<? extends Application> clazz, final Configuration configuration) {
+        try {
+            Application application = clazz.getDeclaredConstructor().newInstance();
```

Comment:
Instantiation in a static method cannot be overridden by an implementation.  I assume the instantiation should better be handled there, possibly by the DI. Would it be better to add an extra method `RuntimeDelegate#bootstrap(final Class<? extends Application> application, final SeBootstrap.Configuration configuration)`?

Reply:
Yes, I'm glad you brought this up. I was unsure about that instantiation. Will update the PR.